scripts/develop.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
 
-# Usage: ./develop.sh [--agpl]
+# Usage: ./develop.sh [--agpl] [--with-tasks]
```

Comment:
Honestly could we just always upload this template and pull the image, rather than having a flag? Seems like it would be a better experience

Reply:
Sure, happy to make it the default